tests/testthat/test-markdown-link.R

```diff
@@ -94,16 +94,19 @@ test_that("short and sweet links work", {
     foo <- function() {}")[[1]]
   expect_equivalent_rd(out1, out2)
 
-  out1 <- roc_proc_text(rd_roclet(), "
+  expect_warning(
+    out1 <- roc_proc_text(rd_roclet(), "
     #' Title
     #'
-    #' See [pkg::function()], [pkg::object].
+    #' See [11pkg::function()], [11pkg::object].
```

Comment:
Would it be better to replace these with actual functions/objects?

Reply:
Hmmm. This is the test for the warnings we get for non-existing packages and topic. So I wanted to make sure that they indeed don't exist.

Reply:
Oooh, makes sense.